src/Interpreters/Cache/QueryConditionCache.h

```diff
@@ -0,0 +1,65 @@
+#pragma once
```

Comment:
For my understanding: Can there be a cache entry with a composite condition, e.g. "x > 3 AND x < 10"? or would we have two cache entries for such conditions? If composite conditions are split ... how are they split? E.g. along ANDs or along ORs?